code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -60,32 +64,36 @@
 
 /obj/item/organ/body_egg/changeling_egg/egg_process(delta_time, times_fired)
 	// Changeling eggs grow in dead people
-	time += delta_time
+	time += delta_time * 10
 	if(time >= EGG_INCUBATION_TIME)
 		pop()
-		Remove(owner.loc)
+		var/turf/remove_loc = get_turf(owner)
+		if(!remove_loc)
+			remove_loc = get_turf(src)
+		Remove(remove_loc)
 		qdel(src)
 
 /obj/item/organ/body_egg/changeling_egg/proc/pop()
-	var/mob/living/carbon/spawned_monkey = new(owner)
-	spawned_monkey.set_species(/datum/species/monkey)
+	var/turf/spawn_loc = get_turf(owner)
```

Comment:
stupid superfluous location code, but can we really trust organs are where they are supposed to be at the codebases current state?

No. We cannot.